tests/pthread/test_pthread_gcc_atomic_fetch_and_op.cpp

```diff
@@ -47,37 +47,37 @@ void *thread_fetch_and_sub(void *arg)
 	pthread_exit(0);
 }
 
-volatile int fetch_and_or_data = 0;
+volatile long fetch_and_or_data = 0;
 void *thread_fetch_and_or(void *arg)
 {
 	for(int i = 0; i < 10000; ++i)
-		__sync_fetch_and_or((int*)&fetch_and_or_data, (int)arg);
+		__sync_fetch_and_or((long*)&fetch_and_or_data, (long)arg);
```

Comment:
Are the casts of the first argument here needed?   (seems like `&fetch_and_or_data` is already `long*`)